treerec/tskit/text_input.c

```diff
@@ -374,7 +379,7 @@ mutation_table_load_text(tsk_mutation_table_t *mutation_table, FILE *file)
         if (err < 0 || *start != '\0') {
             goto out;
         }
-        ret = tsk_mutation_table_add_row(mutation_table, site, node, parent,
+        ret = tsk_mutation_table_add_row(mutation_table, site, node, time, parent,
```

Comment:
This appears to be a serious bug, the fix for which needs to get pushed back to tskit.  `tsk_mutation_table_add_row()` takes parameters `tsk_mutation_table_t *self, tsk_id_t site, tsk_id_t node,
    tsk_id_t parent, double time, ...`, but here they are being passed in the wrong order: `time, parent` instead of `parent, time`.  I'm surprised that nobody else's compiler gave a warning about this error!

Reply:
Gah, I swear I double checked these things N times. And yes - it's very strange I'm not getting these compiler warnings.

But - I don't think there's a change to tskit? This file (`text_input.c`) is not part of tskit; that's the one we wrote locally because we didn't want to go to the trouble of releasing a bulletproof text parser for tskit.

Reply:
This one should really have given you a warning, it's a bit alarming that it didn't.

Right, good, no upstream change in tskit.